shm/alcohol.cpp

```diff
@@ -0,0 +1,38 @@
+#include "alcohol.hpp"
+
+#include <string>
+
+constexpr size_t maxPower{96};
+
+// Override from Cargo
```

Comment:
Leave this comment only in .hpp here is redundant. You can mark which functions are static in cpp.